enumfields/fields.py

```diff
@@ -89,7 +90,20 @@ def __init__(self, enum, **options):
 
     def contribute_to_class(self, cls, name):
         super(EnumFieldMixin, self).contribute_to_class(cls, name)
+
+        def update_initial_field(instance, created, update_fields, **kwargs):
+            if not created:
+                fields = (
+                    instance._meta.get_fields() if update_fields is None
+                    else [getattr(instance, field_name, None) for field_name in update_fields]
+                )
+                for field in fields:
```

Comment:
noo this is too much complicated

Reply:
for me
```
def update_initial_field(instance, created, update_fields, **kwargs):
    initial_field_name = self.get_initial_cache_name()
    instance.__dict__[initial_field_name] = getattr(instance, self.name)
```

Reply:
self will be get from method, this is closure

Reply:
good. this is shorter. I just added the condition to check update_fields otherwise it would be failing on

Reply:
when the fields was not updated into db